react/game/src/app/index.js

```diff
@@ -3,14 +3,20 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { actionCreators as loginActions } from '@redux/auth/actions';
 import Router from '@components/Router';
+import msg from '@constants/messages';
+import withLoading from '@components/withLoading';
 
 class App extends Component {
   componentDidMount() {
     this.props.initApp();
   }
   render() {
     const { appLoading } = this.props;
-    return !appLoading ? <Router /> : <div>Loading</div>;
+    const external = true;
+    const WithLoading = withLoading(Router, external);
```

Comment:
U should use the HOC once, at the `export default ...`

Reply:
ok ok 👌🏼